src/components/organisms/ConfirmedCaseVisual.js

```diff
@@ -86,47 +86,9 @@ export default function ConfirmedCaseVisual(props) {
   )
 
   const isMobile = useMediaQuery({ maxWidth: 960 })
```

Comment:
Is it possible to use `${bps.down("md")}`?

Reply:
@yshing Except this.

Reply:
I will skip this first 🐱

Reply:
@cswbrian No, It's not CSS it's inside JS but we can 
```suggestion
  const isMobile = useMediaQuery({ maxWidth: bps.values.md })
```